Formula/automysqlbackup.rb

```diff
@@ -28,6 +28,8 @@ def caveats; <<-EOS.undent
     EOS
   end
 
+  plist_options :startup => true
```

Comment:
You can omit `:startup => true` and instead specify `:manual` e.g. `:manual => "automysqlbackup"`